R/ecoretriever.R

```diff
@@ -106,6 +106,7 @@ fetch = function(dataset, quiet=TRUE){
 #'
 #' @param dataset the name of the dataset that you wish to download
 #' @param path the path where the data should be downloaded to
+#' @param sub_dir if true the downloaded dataset is stored in a subdirectory of the working directory, defaults to false.
```

Comment:
"if true the downloaded dataset is stored in a subdirectory of the working directory, defaults to false." ->
"if true and the downloaded dataset is stored in subdirectories those subdirectories will be preserved in the working directory, defaults to false."